src/main/java/uk/gov/hmcts/ccd/endpoint/std/DraftsEndpoint.java

```diff
@@ -113,6 +117,7 @@ public DraftResponse updateDraftForCaseWorker(
     @ApiResponses(value = {
         @ApiResponse(code = 200, message = "A displayable draft")
     })
+    @LogAudit(operationType = OperationType.CREATE_CASE)
```

Comment:
UPDATE_CASE?

Reply:
We will ignore drafts totally. We use UPDATE_CASE when we already have the case created (it has a caseId/reference)